File: 01-JavaScript/08-fizzbuzz.js
const prompt = require("prompt-sync")({ sigint: true });

const output = [];

function fizzBuzz() {

    for (count = 1; count <= 5; count++) {
        const myNumber = prompt("Write a hole number: ");
        output.push(myNumber);
    }

    console.log("Your numbers: " + output.join(", "));

    const myNumArray = [];
    output.forEach(item => {
        let myOutput = "";
        if (item % 3 == 0) { myOutput += "Fizz"; }
        if (item % 5 == 0) { myOutput += "Buzz"; }
        if (myOutput == "") { myOutput = item; }

        myNumArray.push(myOutput);
    })

    console.log("\nNumbers replaced with Fizz/Buzz: " + myNumArray.join(", "));

};

fizzBuzz();

File: 02-Python/08-fizzbuzz.py
#!/usr/bin/python

your_numbers = []

def fizz_buzz ():

    repeat = 5
    for _ in range(repeat):
        your_number = int(input("Write a hole number: "))
        your_numbers.append(your_number)
    

    print("\nYour numbers:\n " +  str(your_numbers))

    num_fizz_buzz = []
    for num in range(0, len(your_numbers)):
        fizz_buzz = ""
        if(your_numbers[num] % 3 == 0):
            fizz_buzz += "Fizz"
        if(your_numbers[num] % 5 == 0):
            fizz_buzz += "Buzz"
        if fizz_buzz == "":
            fizz_buzz = your_numbers[num]

        num_fizz_buzz.append(fizz_buzz)
    
    print("\nNumbers replaced with Fizz/Buzz: \n" + str(num_fizz_buzz))


fizz_buzz()
